src/main/java/com/buddy/api/domains/adoption/entities/AdoptionPostFollowUpEntity.java

```diff
@@ -0,0 +1,64 @@
+package com.buddy.api.domains.adoption.entities;
+
+import com.buddy.api.domains.adoption.enums.AdaptationStatusEnum;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+@Data
+@Builder
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+@Table(name = "adoption_post_follow_up")
+public class AdoptionPostFollowUpEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    @EqualsAndHashCode.Include
+    @Column(name = "follow_up_id", nullable = false, unique = true)
+    private UUID followUpId;
+
+    @ManyToOne(fetch = FetchType.LAZY, optional = false)
+    @JoinColumn(name = "request_id", referencedColumnName = "request_id", nullable = false)
+    @ToString.Exclude
+    private AdoptionRequestV2Entity adoptionRequest;
+
+    @Column(name = "visit_date")
+    private LocalDate visitDate;
+
+    @Column(name = "report")
+    private String report;
+
+    @Column(name = "adaptation_status")
```

Comment:
```suggestion
    @Column(name = "adaptation_status", nullable = false)
```

Sejam quais forem os status que decidirmos colocar, um pedido de adoção sempre vai estar em um status. Imagino que você tenha feito assim para poder prosseguir com a implementação sem ter os status definidos, correto? Que tal criarmos os status `ON_GOING`, `APPROVED` e `REJECTED`, só pra começar, enquanto não definimos isso melhor?

Reply:
Estava pensando sobre esse status e cheguei nos seguintes:

**NOT_ADAPTED**: O animal não se adaptou ao novo ambiente e ainda está apresentando dificuldades.
**PARTIALLY_ADAPTED**: O animal está se adaptando, mas ainda enfrenta desafios ou não está totalmente confortável.
**FULLY_ADAPTED**: O animal se adaptou completamente ao novo lar e apresenta comportamentos normais.
**IN_PROGRESS**: O processo de adaptação ainda está em andamento, podendo levar mais tempo.
**REQUIRES_ATTENTION**: O animal está tendo dificuldades significativas e pode precisar de suporte ou intervenções adicionais.
**REJECTED**: O processo de adoção não foi bem-sucedido, e o animal foi devolvido ao abrigo ou à organização de adoção.
**OTHER**: Um campo genérico para situações que não se encaixam nas outras categorias.

O que acha?

Reply:
Gostei!